### PR TITLE
Link the JSON library using vcpkg in test files

### PR DESCRIPTION
**Pull request by shlomnissan**
**Summary**

This PR removes the `nlohmann/json` header file from our test directory and
links it to the library imported through vcpkg. Additionally, it updates the
`README` file, making the vcpkg installation instructions more concise.

**Test Plan**

- All existing tests have passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,36 +14,13 @@ This driver uses the following libraries:
...
-```
-
-If you've built the driver before, consider deleting your build directory first. If you encounter any problems, refer to the [source vcpkg guide](https://learn.microsoft.com/en-us/vcpkg/get_started/get-started?pivots=shell-bash) or open an issue for help.
+- `git clone https://github.com/microsoft/vcpkg.git`
+- `cd vcpkg && ./bootstrap-vcpkg.sh `
+- Add export commands to your shell's profile script (e.g., `~/.bashrc` or `~/.zshrc`)
+  - `export VCPKG_ROOT=/path/to/vcpkg`
+  - `export PATH=$VCPKG_ROOT:$PATH`
+- Select the default CMake preset before building (consider deleting your build directory first)
+  - `cmake --preset=default`
```
Comment (aliddell) [inline README.md:23]: At least in CLion, I've had to manually specify the target triplet as well. Here's what I get when I don't:

```
[60/108] Linking CXX shared module src\acquire-driver-zarr.dll
FAILED: src/acquire-driver-zarr.dll 
...
LINK : fatal error LNK1181: cannot open input file 'blosc_static.lib'
```

File: README.md [hunk 2]
```diff
@@ -14,36 +14,13 @@ This driver uses the following libraries:
...
-```
-cmake --preset=default
-```
-
-If you've built the driver before, consider deleting your build directory first. If you encounter any problems, refer to the [source vcpkg guide](https://learn.microsoft.com/en-us/vcpkg/get_started/get-started?pivots=shell-bash) or open an issue for help.
+- `git clone https://github.com/microsoft/vcpkg.git`
+- `cd vcpkg && ./bootstrap-vcpkg.sh `
+- Add export commands to your shell's profile script (e.g., `~/.bashrc` or `~/.zshrc`)
+  - `export VCPKG_ROOT=/path/to/vcpkg`
+  - `export PATH=$VCPKG_ROOT:$PATH`
```
Comment (aliddell) [inline README.md:21]: The process is [somewhat different](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_environment_variables?view=powershell-7.4#set-environment-variables-in-the-system-control-panel) on Windows. Could you add a note?

File: README.md (hunk 1 above)
Comment (shlomnissan) [reply README.md:23]: Are you building the project on Windows? This error indicates that the linker is unable to locate the static library. Have you used the windows static triplet (`x64-windows-static`)?

File: README.md (hunk 2 above)
Comment (shlomnissan) [reply README.md:21]: Sure, I can add a note later today.

File: README.md (hunk 1 above)
Comment (aliddell) [reply README.md:23]: Yeah, this is exactly what I mean. If I don't specify the target triplet on Windows, I can't build. It should be part of the instructions.

Comment (shlomnissan) [reply README.md:23]: Ahh I see, I read this early in the morning and I missed the (”what I get if I don’t part”) 😂 I'll add it to the instructions.

**Review by aliddell — APPROVED** [review]
😎

**Pull request merged**